docs/about/whats_new.rst

```diff
@@ -8,30 +8,65 @@ What's New
 Next Version
 ============
 
-- docs: fix broken links :pull:`1725`
-- Update dependencies :pull:`1727`, :pull:`1731`
+v1.9.3 (15th April 2025)
+========================
+
+Contains a fix to a serious bug affecting dask usage, preparation for psycopg3, and a lot of other minor cleanup,
```

Comment:
Should perhaps #1670 go into this release? It seems reasonable to make that type of switch early in the 1.9 series.

Reply:
Done.